recipes/cobra/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: cobra
+  version: "0.4.1"
+
+source:
+  fn: cobra-0.4.1.tar.gz
+  url: https://pypi.python.org/packages/source/c/cobra/cobra-0.4.1.tar.gz
+  md5: a96799117abed427333bb57e007a8b82
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - glpk
+
+  run:
+    - python
+    - six
+    - numpy
+    - scipy
+    - lxml
+    #- python-libsbml
```

Comment:
For this particular library, it's a little bit of a pain unfortunately.
That recipe required a patch to work.

On Tue, Mar 22, 2016 at 4:06 PM, jakirkham <notifications@github.com> wrote:

> In recipes/cobra/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/182#discussion_r57085314>
> :
>
> > +  number: 0
> > +
> > +requirements:
> > +  build:
> > +    - python
> > +    - setuptools
> > +    - six
> > +    - glpk
> > +
> > +  run:
> > +    - python
> > +    - six
> > +    - numpy
> > +    - scipy
> > +    - lxml
> > +    #- python-libsbml
>
> Alright, maybe we can just include Mac and Linux builds if you have
> something. Though it can totally be a separate PR.
>
> I haven't used this so wasn't sure what went into getting it built. I was
> hoping python-* meant easy. [image: :wink:]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/182/files/692eb5240589a6ab0a1c19b6b920769ada78043d#r57085314>
>